### PR TITLE
[1.1] Add NUGET_FEED_URL to env-vars in build pipelines

### PR DESCRIPTION
**Pull request by ianhays**
The 1.1 pipebuilds were not updated with the secret flushing a few months back, so publishing and finalization steps are getting skipped because of missing environment variables.

@MattGal and @eerhardt PTAL.

### CODE REVIEW

File: buildpipeline/Core-Setup-CentOS-x64.json [hunk 1]
```diff
@@ -166,7 +166,7 @@
       "allowOverride": true
     },
     "BuildArguments": {
-      "value": "--skip-prereqs --configuration $(BuildConfiguration)  --docker centos.7 --targets Default --env-vars NUGET_API_KEY=$(NUGET_API_KEY),GITHUB_PASSWORD=$(GITHUB_PASSWORD),REPO_PASS=$(REPO_PASS),CONNECTION_STRING=$(CONNECTION_STRING)",
+      "value": "--skip-prereqs --configuration $(BuildConfiguration)  --docker centos.7 --targets Default --env-vars NUGET_FEED_URL=$(NUGET_FEED_URL), NUGET_API_KEY=$(NUGET_API_KEY),GITHUB_PASSWORD=$(GITHUB_PASSWORD),REPO_PASS=$(REPO_PASS),CONNECTION_STRING=$(CONNECTION_STRING)",
```
Comment (eerhardt) [inline buildpipeline/Core-Setup-CentOS-x64.json]: You can't have spaces in the `--env-vars` list.

`NUGET_FEED_URL=$(NUGET_FEED_URL), NUGET_API_KEY=$(NUGET_API_KEY),GITHUB_PASSWORD=$(GITHUB_PASSWORD)`

**Comment by MattGal** [comment]
Seems OK except for the space issue;  however given that this specific value isn't a secret it shouldn't have been a problem either (i.e. the Debian 9 problem may be the only one).  Does not hurt at all to be explicit, though.

**Comment by ianhays** [comment]
This resolved the error for whatever reason. I've got a pipebuild going against this PR and it is publishing to myget: https://devdiv.visualstudio.com/DevDiv/Net%20CoreFX/_build/index?buildId=1729031&_a=summary

**Comment by ianhays** [comment]
It looks like the only publishing completed is being done by the ubuntu1804 build. When I look at e.g. the Fedora28 build FinalizeBuild is still empty and when I look at the Fedora27 build it is trying to push an OSX package (which was already uploaded by the ubuntu1804 build!)

**Pull request merged**